modules/mono/mono_gd/gd_mono_class.cpp

```diff
@@ -139,6 +139,20 @@ void GDMonoClass::fetch_attributes() {
 	attrs_fetched = true;
 }
 
+void GDMonoClass::fetch_method_list() {
+
+	if (method_list_fetched)
+		return;
+
+	void *iter = NULL;
+	MonoMethod *raw_method = NULL;
+	while ((raw_method = mono_class_get_methods(get_mono_ptr(), &iter)) != NULL) {
+		method_list.push_back(memnew(GDMonoMethod(mono_method_get_name(raw_method), raw_method)));
+	}
+
+	method_list_fetched = true;
+}
```

Comment:
This may as well be moved to `get_all_methods` since that's the only place where it's used.

Reply:
Done.